packages/wpcom.js/src/lib/site.post.js

```diff
@@ -206,40 +201,6 @@ class SitePost {
 		const comment = new Comment( null, this._id, this._sid, this.wpcom );
 		return comment.replies( query, fn );
 	}
-
-	/**
-	 * Create a `Like` instance
-	 * @returns {Like} Like instance
-	 */
-	like() {
-		return new Like( this._id, this._sid, this.wpcom );
-	}
-
-	/**
-	 * Create a `Reblog` instance
-	 * @returns {Reblog} Reblog instance
-	 */
-	reblog() {
```

Comment:
Is it worth to keep maintaining History.md with these changes?

https://github.com/Automattic/wp-calypso/blob/37c161b6d44040aef789e62f7f199d27c6824806/packages/wpcom.js/History.md#L3

Reply:
Yes, I just added a missing `History.md` entry.